packages/components/tsup.config.ts

```diff
@@ -1,3 +1,4 @@
+import * as fs from 'fs/promises';
```

Comment:
why add?
```suggestion
```

Reply:
Accident. Thanks for catching it.
I was thinking about moving the fonts here, and copying them to `dist`, and this is a leftover.

Just removed it.